bessctl/bessctl

```diff
@@ -141,14 +142,15 @@ def main():
         line_buf = []
 
         for arg in sys.argv[1:]:
+            arg = arg.decode()
```

Comment:
The elements of `sys.argv` should already be strings (not bytes) in py3k.  Are you sure this is needed?